README.md

```diff
@@ -68,6 +106,15 @@ module.exports = {
       ],
     },
   },
+
+  // Alias for `mocha.timeout`.
```

Comment:
I guess these aliases exist due to historical reasons and were left here for compatibility with old configs. Let's not document them to prevent misunderstanding?